mission_control/navigator_launch/launch/subsystems/tf.launch

```diff
@@ -1,18 +1,18 @@
 <launch>
-    <node pkg="tf" type="static_transform_publisher" name="measure_point_tf" args="1.2319 0 0  0 0 0 1  /base_link /measurement 100" /> <!-- Very front center of top part -->
+    <node pkg="tf" type="static_transform_publisher" name="measure_point_tf" args="1.2319 0 0  0 0 0 1  /base_link /measurement 10" /> <!-- Very front center of top part -->
     <node pkg="tf" type="static_transform_publisher" name="ins_tf" args="-0.652272 0.136525 0.192278  0 0 0 1  /measurement /ins 10" />
     
     <!-- LIDAR -->
-    <node pkg="tf" type="static_transform_publisher" name="lidar_tf" args="-0.277622 0 0.648208 0 0 0 1  /measurement /velodyne 100" />
-    <node pkg="tf" type="static_transform_publisher" name="sick_tf" args="0.5334 -0.0254 -0.6858  0 0 0 1  /measurement /sick 100" />
+    <node pkg="tf" type="static_transform_publisher" name="lidar_tf" args="-0.277622 0 0.648208 0 0 0 1  /measurement /velodyne 10" />
+    <node pkg="tf" type="static_transform_publisher" name="sick_tf" args="0.5334 -0.0254 -0.6858  0 0 0 1  /measurement /sick 10" />
 
     <!-- Cameras -->
-    <node pkg="tf" type="static_transform_publisher" name="stereo_tf" args="0.017272 -0.100076 0.19812 0.56679 -0.56679 0.42278 -0.42278 /measurement /stereo_front 100" />
```

Comment:
@DSsoto chose to remove stereo_tf frame and position each camera individually since fudging their TF as a group gave us subpar results (meaning measurements are not perfect, or the cameras don't sit in the same spot in the tube)